lib/api/utils/transform-response.js

```diff
@@ -0,0 +1,30 @@
+import URI from 'urijs';
+
+// make these available for eval
+import _faker from 'faker/build/build/faker';
+import _state from 'api/local-state';
+
+export function transformResponse(responseRule, request) {
+    // Setup local variables for eval
+  let url = URI(request.url);
+  let path = url.segmentCoded();
+  let query = url.search(true);
+  let data = JSON.parse(request.body);
```

Comment:
Please don't assume the response is JSON. It can be many other responses.

I would prefer if it would have worked something more similar to angular templating language which means it doesn't care in what context it runs as long as there is the binding symbols {{ }}

This feature previously failed when the responses were different than what you expect usually.